src/main/java/com/teamcautionrobotics/robot/ExpanderHatch.java

```diff
@@ -0,0 +1,26 @@
+
+package com.teamcautionrobotics.robot;
+
+import edu.wpi.first.wpilibj.*;
+
+public class ExpanderHatch{
```

Comment:
Add a space before the bracket.
```suggestion
public class ExpanderHatch {
```